src/data/fintype/basic.lean

```diff
@@ -43,13 +92,13 @@ fintype.complete x
 by ext; simp
 
 lemma univ_nonempty_iff : (univ : finset α).nonempty ↔ nonempty α :=
-by rw [← coe_nonempty, coe_univ, set.nonempty_iff_univ_nonempty]
+by rw [←coe_nonempty, coe_univ, set.nonempty_iff_univ_nonempty]
 
 lemma univ_nonempty [nonempty α] : (univ : finset α).nonempty :=
 univ_nonempty_iff.2 ‹_›
 
 lemma univ_eq_empty : (univ : finset α) = ∅ ↔ ¬nonempty α :=
-by rw [← univ_nonempty_iff, nonempty_iff_ne_empty, ne.def, not_not]
+by rw [←univ_nonempty_iff, nonempty_iff_ne_empty, ne.def, not_not]
```

Comment:
AFAICT a space after `←` is a lot more popular than without, a quick search 5599 versus 2261 lines respectively. Or do we have a style guideline saying to prefer `←` without the space?

If it's not in the style guide, then I'm not going to block this PR over those changes. Just beware that it is going contrary to the trend.